README.md

```diff
@@ -8,10 +8,14 @@ Upstream documentation can be found at https://github.com/GoogleCloudPlatform/sp
 ## Usage
 
 The Spark Operator may be deployed using the Juju command line as follows
+
 ```bash
-juju deploy spark-k8s
+juju deploy spark-k8s spark
```

Comment:
Please let me know if this is also affecting deployments that are not necessarily the upstream examples, if not, I think we can leave the Usage section as it is and add an Examples one with specific instructions for running the examples.

## Usage
```bash
juju deploy spark-k8s
```

## Examples
Running upstream examples (add link to upstream examples)
1. Deploy `spark-k8s` as `spark` due to the upstream examples requiring \<add longer explanation\>
`juju deploy spark-k8s spark`
2. Run examples \<add instructions to run examples\>
` commands to run examples`

Reply:
The key is that the service role name is the same as the application name.

The examples from canonical/spark-k8s are used for integration tests, integration tests deployment is done as `spark` so if we change the example then the integration test will stop working.

On the other hand, what is the benefit of having a different name for the application and the service account?

Reply:
Updating the README with those instructions does not affect the examples or our integration tests. I'd rather have a separate set of instructions where we specify why we are deploying the charm as `spark` for running the examples as for other purposes we could just deploy the charm as `spark-k8s` with no problem.

> On the other hand, what is the benefit of having a different name for the application and the service account?

It won't be different, juju will name the sa the same as the charm.